ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/ws/controller/RevocationListControllerV2.java

```diff
@@ -56,13 +59,22 @@ public RevocationListControllerV2(RevokedCertDataService revokedCertDataService)
     @CrossOrigin(origins = {"https://editor.swagger.io"})
     @GetMapping(value = "/revocationList")
     public @ResponseBody ResponseEntity<RevocationResponse> getRevokedCerts(
+            WebRequest request,
             @RequestParam(required = false, defaultValue = "0") Long since)
             throws HttpStatusCodeException {
         Instant now = Instant.now();
         List<DbRevokedCert> revokedCerts =
                 revokedCertDataService.findReleasedRevokedCerts(since, now);
         List<String> revokedUvcis =
                 revokedCerts.stream().map(DbRevokedCert::getUvci).collect(Collectors.toList());
+        String certEtag = EtagUtil.getUnsortedListEtag(false, revokedCerts.stream().map(Object::toString).collect(
```

Comment:
`Object::toString` seems a bit unsafe to me. if someone was to implement the toString method, not including all fields of `DbRevokedCert` this would fail. i assume you're interested in the hash code. in that case i would map to the hashcode

`revokedCerts.stream().map(c -> Integer.toHexString(c.hashCode())).collect(Collectors.toList())`